tests/other_test.go

```diff
@@ -19,3 +21,28 @@ func TestRunDefaultExample(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
+
+func TestRunFSCloudExample(t *testing.T) {
```

Comment:
I do not see any fscloud example [here](https://github.com/terraform-ibm-modules/terraform-ibm-event-streams/tree/kms-feat/examples), this will not work.

Reply:
didn't notice, added back the example